bedrock/pocket/templates/pocket/pocket-and-firefox.html

```diff
@@ -25,21 +25,21 @@
   media_class='mzp-l-split-h-center',
   image_url='img/pocket/save-inspirations.png',
   )%}
-  <span class="tagline">Pocket for Firefox</span>
```

Comment:
Issue (blocking): page title above is untranslated:

`{% block page_title %}for Firefox{% endblock %}`